packages/perseus-editor/src/widgets/plotter-editor.tsx

```diff
@@ -89,30 +71,8 @@ const formatNumber = (num) => "$" + knumber.round(num, 2) + "$";
 class PlotterEditor extends React.Component<Props, State> {
     static widgetName = "plotter" as const;
 
-    static defaultProps: DefaultProps = {
-        ...editorDefaults,
-        correct: [1],
-        starting: [1],
-
-        type: "bar",
-        labels: ["", ""],
-        categories: [""],
-
-        picSize: 30,
-        picBoxHeight: 36,
-        plotDimensions: [275, 200],
-        labelInterval: 1,
-
-        // @ts-expect-error - TS2322
-        get picUrl() {
-            const staticUrl = Dependencies.getDependencies().staticUrl;
-            if (staticUrl) {
-                return staticUrl("/images/badges/earth-small.png");
-            }
-
-            return null;
-        },
```

Comment:
I just felt like this was a little much just to have a placeholder image.

Reply:
I've always really disliked that default props has behaviour embedded in it via a getter! 

What does the Plotter show by default in the editor with this now defaulting to `null`.

Reply:
Turns out this was kind of a hard question to answer.

1. I added a story, but SB didn't properly resolve the URL for the image; it looked the same in `main` and this branch
2. I tried to look in the content editor, but Plotter is a hidden widget so I couldn't make new widget to test

I think I might just merge this PR since the widget is hidden anyway. Shouldn't change the editing experience since they'll never see an empty Plotter widget right?

Reply:
> I think I might just merge this PR since the widget is hidden anyway. Shouldn't change the editing experience since they'll never see an empty Plotter widget right?

Good point!  For hidden widgets, they can never be "created" in the editor anyways, so defaults for these widgets feel like a fading concern.